### DISCUSSION TITLE
How to use Modal pop-up and confirm pop-ups widgets

### DISCUSSION THREAD

**Discussion by Magnetoabhi** [breezefront/community#52] [1. Help] [answered]
I am new to the Breeze theme and am using breeze evolution theme, and I need to add modal popup in one of my module can you please advice me on how to use it in breeze theme with example

**Comment by vovayatsyuk** [comment, marked as answer]
Static html example:

```
<button class="button primary modal-opener">Show popup</button>

<div style="display:none" data-mage-init='{
        "Magento_Ui/js/modal/modal": {
            "trigger": ".modal-opener"
        }
    }'>
    Modal content
</div>
```

JS example:

```
$('.selector').modal();
$('.selector').modal('openModal');
```

**Reply by Magnetoabhi** [reply]
> Static html example:
> 
> ```
> <button class="button primary modal-opener">Show popup</button>
> 
> <div style="display:none" data-mage-init='{
>         "Magento_Ui/js/modal/modal": {
>             "trigger": ".modal-opener"
>         }
>     }'>
>     Modal content
> </div>
> ```
> 
> JS example:
> 
> ```
> $('.selector').modal();
> $('.selector').modal('openModal');
> ```

This was really helpfull!!

**Comment by Magnetoabhi** [comment]
How to use it in script in the phtml file itself with dependency?

In luma based we can use the following but how can I use it in breeze
```
<script>
require([
        'jquery',
        'Magento_Ui/js/modal/confirm'
    ],
    function($, confirmation) {
        $('#accept_user').on('click', function (e){
            e.preventDefault();
            confirmation({
                title: 'Accept user',
                content: 'Do you want to accept this user?',
                actions: {

                    confirm: function () {
                        <?php /* @escapeNotVerified */ echo $block->activateUser('$userId'); ?>
                    },

                    cancel: function () {
                        return false;
                    }
                }
            });
        });
    });
</script>
```
